src/components/renderer/form-masked-input.vue

```diff
@@ -2,6 +2,7 @@
   <div class="form-group">
     <label v-uni-for="name">{{ label }}</label>
     <component
+      v-if="componentType!=='input'"
```

Comment:
That’s  because Vue binds input v-model in a different way than in a component. https://github.com/vuejs/vue/issues/7393

Reply:
Ah, good to know! It looks like it will still work if you bind the props individually using `v-bind`. It's mentioned in the docs at https://vuejs.org/v2/guide/components.html#Dynamic-Components:

> For some properties such as `value` to work as you would expect, you will need to bind them using the `.prop` modifier.

And in this issue: https://github.com/vuejs/vue/issues/10444.

So instead of `v-model`, you could have `:value.prop="localValue" @input="localValue = $event.target.value"`. The event listener will need to have slightly different logic to handle the DOM event vs `v-model`, but I think that having the branching there will still eliminate a lot of duplication between the dynamic component and input, which might make it easier to update in the future. 

The thing I want to guard against is someone coming by and modifying the props on the dynamic component, but leaving the input the same. This could introduce subtle bugs, especially since this component may not be well (or at all) tested.

An alternative would also be to pass down the props as an object, so you could have something like this:

```html
<component
  v-if="componentType !== 'input'"
  :is="componentType"
  v-model="localValue"
  v-bind="componentProps"
/>
<input v-else
  v-model="localValue"
  v-bind="componentProps"
>
```

And in the data object:
```js
componentProps: {
  componentConfig: 'someValue'
  name: 'someValue'
  name: 'someValue'
  formControl: 'someValue'
  classList: 'someValue'
  text: 'someValue'
}
```